src/libcollectionstest/slice.rs

```diff
@@ -867,17 +867,6 @@ fn test_vec_default() {
 }
 
 #[test]
-fn test_bytes_set_memory() {
```

Comment:
Huh I thought I removed the trait, but I think we actually *just* deprecated the trait so I got mixed up.